### PR TITLE
Update hadith API

### PR DESCRIPTION
**Pull request by baaki20**

### CODE REVIEW

File: .env
Comment (oovaa) [inline .env:1]: will vscode use your env file ?
check this online or ask the support team

File: hasanah/connect.js
Comment (oovaa) [inline hasanah/connect.js:1]: have you checked if vscode supports .mjs files ?

sorry for asking to many questions but this extension is important for me and we are both learning new things

File: hasanah/hadith.mjs
Comment (oovaa) [inline hasanah/hadith.mjs:1]: is my previous function working well if the language is set to Arabic ?

we need to use the two apis

Comment (oovaa) [reply hasanah/hadith.mjs:1]: you can create a separate English hadith file and import the ready function

**Review by oovaa — CHANGES_REQUESTED** [review]
good job my friend but we need some work to improve it

File: .env
Comment (baaki20) [reply .env:1]: I searched online and the answer was no. That we will need to configure the extension to access the variable stored in it

File: hasanah/connect.js
Comment (baaki20) [reply hasanah/connect.js:1]: So far I haven't encountered any problems but I'll try to change it back to .js.
The reason why the file was stored as .mjs was that, I was encountering some errors, the errors vanished after I changed the file extension to .mjs

File: hasanah/hadith.mjs
Comment (baaki20) [reply hasanah/hadith.mjs:1]: > is my previous function working well if the language is set to Arabic ?
> 
> we need to use the two apis

Yes, it was working perfectly.

**Pull request closed without merging**